src/PaymentMethod.js

```diff
@@ -212,78 +258,87 @@ export default class PaymentMethod extends Component {
+                <CreditCardForm
+                  ref={this.props.saveRef}
+                  account={this.state.accountModel}
+                  setAccount={(accountProperty, field, value) =>
+                    this.setAccount(accountProperty, field, value)
+                  }
+                  showACHLink={this.isACHEnabled()}
+                  changePaymentMethod={() => this.changePaymentMethodACH()}
+                  {...subProperties}
+                >
+                  {renderCardForms}
+                </CreditCardForm>
+              </div>
+            }
+            {
+              method === PaymentMethods.METHOD_ACH &&
+              <div id="bank-info">
+                <AchForm
+                  ref={this.props.saveRef}
+                  hideTogglePlaid={this.isPlaidEnabled() === true ?
+                    false : true
+                  }
+                  setAccount={(accountProperty, field, value) =>
+                    this.setAccount(accountProperty, field, value)
+                  }
+                  changePaymentMethod={() => this.changePaymentMethodCC()}
+                  showCardLink={this.isCardEnabled()}
+                  togglePlaidHandler={this.togglePlaidHandler}
+                  {...subProperties}
+                >
+                  {renderAchForms}
+                </AchForm>
+              </div>
+            }
+            {
+              method === PaymentMethods.METHOD_PLAID &&
+              <div id="bank-info">
+                <PlaidForm
+                  ref={this.props.saveRef}
+                  account={this.state.accountModel}
+                  setAccount={(accountProperty, field, value) =>
+                    this.setAccount(accountProperty, field, value)
+                  }
+                  changePaymentMethod={() => this.changePaymentMethodCC()}
+                  togglePlaidHandler={this.togglePlaidHandler}
+                  {...subProperties}
+                />
+              </div>
+            }
+            {method === false && <div className="ui clearing divider"></div>}
+            {method === false &&
+              <ButtonGroup
+                showAccept={false}
+                onLast={onLast}
+                onCancel={onCancel}
+                hideNext={true}
+                buttonStylesPrimary={this.props.buttonStylesPrimary}
+                buttonStylesSecondary={this.props.buttonStylesSecondary}
               />
-            </div>
-        }
-        {method === false && <div className="ui clearing divider"></div>}
-        {method === false &&
-          <ButtonGroup
-            showAccept={false}
-            onLast={onLast}
-            onCancel={onCancel}
-            hideNext={true}
-            buttonStylesPrimary={this.props.buttonStylesPrimary}
-            buttonStylesSecondary={this.props.buttonStylesSecondary}
-          />
+            }
+          </React.Fragment>
+          : <p>Loading instrument, please wait...</p>
         }
```

Comment:
This is significantly concerning. 

I'm concerned by this loading field, we are creating a bad experience for the user when the developer doesn't customize the CSS to be a fixed height or width.  @flobotro @chris-revops 

Might I recommend that loading be a state that they define as a prop, but default this to always showing empty fields?

Reply:
Discussed, will be customizable at a later date.